pkg/daemon/controller/controller.go

```diff
@@ -488,9 +488,21 @@ func (sc *ServiceController) resolveNodeVariables(svc servicescm.Service) (map[s
 func (sc *ServiceController) resolvePowershellVariables(svc servicescm.Service) (map[string]string, error) {
 	vars := make(map[string]string)
 	for _, script := range svc.PowershellPreScripts {
+		// replaces the values in the PowershellPreScripts path with the node args
+		if len(script.NodeArgs) != 0 {
```

Comment:
you can check if this is 0 and return early here. This will remove indentation.

Reply:
if we return early, then it won't get to the sc.psCmdRunner.Run, and the script won't be executed... This block should only run when NodeArgs are set